modules/lightning_features/lightning_core/modules/lightning_test/lightning_test.install

```diff
@@ -81,21 +81,6 @@ function lightning_test_install() {
       ],
       'region' => 'content',
     ])
-    ->setComponent('field_z_image', [
```

Comment:
All of this was moved into the new upload_bundles_test module, which is a new test-only module that was split out so as not to cause a conflict which prevented the CKEditor media browser from being properly testable when lightning_test is turned on.